pages/Items.md

```diff
@@ -0,0 +1,18 @@
+The Items category contains various items that all have their own unique use.
+
+## Table of Contents
+* [Portable Crafter](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Portable-Crafter)
+* [Portable Dustbin](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Portable-Dustbin)
+* [Rag](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Medical-Supply)
+* [Bandage](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Medical-Supply)
+* [Splint](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Medical-Supply)
+* [Vitamins](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Medical-Supply)
+* [Medicine](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Medical-Supply)
```

Comment:
Since this page will host all medical supplies, it should rather be called "Medical-Supplies", not "Medical-Supply"

Reply:
Resolved.